### PR TITLE
Patch: Scheduler updates

### PR DESCRIPTION
**Pull request by onfire4g05**
- Fixes bug with variables
- Added babel support so public files can be compiled on build to work with old browsers

### CODE REVIEW

**Comment by millsemillse** [comment]
fantastic, great tool! 
one point: <> in the condition does not work with variable values providing additional chars , so -30 dBFS will not work with < or >. Maybe a sorting out by the user in the frontend ( a field with a given pattern abcVALUExyz) could solve this issue.

**Comment by krocheck** [comment]
FYI, the full commit stack is coming through because I did a squash and merge originally.  The same should happen to merge this in if it look OK.

**Comment by onfire4g05** [comment]
Hah! I was wondering why the full stack showed but the changes tab showed correctly. I had to merge this way, since we're currently using the scheduler, so basically I had to fork the dev branch and merge those changes back in so I can still use the master branch. 😬 There will probably be other changes like this in the future; hope that's ok.

As for the issue @millsemillse brought up, if anyone has feedback on the best way for that... I'm thinking the best thing is just doing a `parseFloat()` on the variable if using `<` or `>`.

File: package.json [hunk 1]
```diff
@@ -81,15 +81,16 @@
 			"!font/*",
 			"!tools/*",
 			"!*.md",
-			"!node_modules/sharp/vendor/lib"
+			"!node_modules/sharp/vendor/lib",
+			"!public/js/es6"
```
Comment (Julusian) [inline package.json]: I think this should be `public/js/dist`

File: package.json [hunk 2]
```diff
@@ -81,15 +81,16 @@
...
 		]
 	},
 	"scripts": {
 		"prod": "./tools/build_writefile.sh && ./bitfocus-skeleton/start.sh",
-		"dev": "./tools/build_writefile.sh && DEVELOPER=1 ./bitfocus-skeleton/start.sh",
+		"dev": "rm -rf ./public/js/dist ; ./tools/build_writefile.sh && DEVELOPER=1 ./bitfocus-skeleton/start.sh",
 		"update": "./tools/update.sh",
 		"pack": "electron-builder --dir",
-		"dist": "rm -rf electron-output && electron-builder",
+		"dist": "./node_modules/.bin/babel ./public/js/es6 --out-dir ./public/js/dist ; rm -rf electron-output && electron-builder",
```
Comment (Julusian) [inline package.json]: I think this should this be using `public/js/babel` not `public/js/es6`?

File: public/js/babel/schedule.js [hunk 3]
```diff
@@ -0,0 +1,541 @@
+class schedule_frontend {
```
Comment (Julusian) [inline public/js/babel/schedule.js]: The old public/schedule.js should be deleted as this replaces it

**Review by Julusian — CHANGES_REQUESTED** [review]
It looks like a few paths have gotten messed up.

I am wondering if using babel like this is the way to go or if we should use webpack instead which will itself use babel, but will output us a single combined js file and can do hot-reloading.
@haakonnessjoen what do you think?

File: public/js/babel/schedule.js (hunk 3 above)
Comment (onfire4g05) [reply public/js/babel/schedule.js]: Weird, I renamed the file in commit 460bb72, but it came back in 8b05c5d merge. Not sure how or why that happened.

**Comment by willosof** [comment]
Where are we on this? :)

**Comment by onfire4g05** [comment]
Should all be resolved.

**Pull request merged**